docs/runbook.md

```diff
@@ -13,103 +13,195 @@
   - [tags](#tags)
   - [parent](#parent)
     - [path](#path)
-    - [strategy](#strategy)
   - [extension](#extension)
   - [variable](#variable)
     - [is_secret](#is_secret)
     - [file](#file)
     - [name](#name-1)
-  - [value](#value)
-  - [artifact](#artifact)
-    - [name](#name-2)
-    - [locations](#locations)
-      - [type](#type)
-      - [path](#path-1)
+    - [value](#value)
   - [notifier](#notifier)
     - [console](#console)
+      - [log_level](#log_level)
     - [html](#html)
+      - [path](#path-1)
+      - [auto_open](#auto_open)
   - [environment](#environment)
-    - [name](#name-3)
-    - [topology](#topology)
-    - [nodes](#nodes)
-    - [nodes_requirement](#nodes_requirement)
-      - [type](#type-1)
+    - [environments](#environments)
+        - [name](#name-2)
+        - [topology](#topology)
+        - [nodes](#nodes)
+        - [nodes_requirement](#nodes_requirement)
+          - [type](#type)
   - [platform](#platform)
   - [testcase](#testcase)
+    - [criteria](#criteria)
 
 ## How-to
```

Comment:
This section is intented to provide some samples. You can provide a minimal example for each one here.